vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -263,21 +273,31 @@ void FlyoutShadowNode::updateProperties(const folly::dynamic&& props)
     winrt::FlyoutBase::SetAttachedFlyout(m_targetElement, m_flyout);
   }
 
-  if (updateOffset)
+  if (updateOffset && m_isFlyoutShowOptionsSupported)
   {
     winrt::Point newPoint(m_horizontalOffset, m_verticalOffset);
     m_showOptions.Position(newPoint);
   }
 
-  winrt::Rect exclusionRect = winrt::Rect(100, 100, 20, 20);
-  m_showOptions.ExclusionRect(exclusionRect);
+  if (m_isFlyoutShowOptionsSupported)
+  {
+    winrt::Rect exclusionRect = winrt::Rect(100, 100, 20, 20);
+    m_showOptions.ExclusionRect(exclusionRect);
+  }
 
   if (updateIsOpen)
   {
     if (m_isOpen)
     {
       AdjustDefaultFlyoutStyle();
-      m_flyout.ShowAt(m_targetElement, m_showOptions);
+      if (m_isFlyoutShowOptionsSupported)
```

Comment:
ShowAt(x, showOptions) is introduced in IFlyoutBase5.
I prefer to do cast.
isFlyoutoutShowOptions = !! (m_flyout.try_as<winrt::IFlyoutBase5>());

Reply:
Ok, I've updated the PR with a new commit.